test/MyFirstApiConsume.test.js

```diff
@@ -0,0 +1,82 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+
+describe('First Api Tests', () => {
+
+    //Consumiendo un servicio GET
+    it('Consume GET Service', async () => {
+        const response = await axios.get('https://httpbin.org/ip');
+
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(response.data).to.have.property('origin');
+    });
+
+    //Consumiendo un servicio GET con Query Parameters
+    it('Consume GET Service with query parameters', async () => {
+        const query = {
+            name: 'John',
+            age: '31',
+            city: 'New York'
+        };
+
+        const response = await axios.get('https://httpbin.org/get', { query });
+
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(response.config.query).to.eql(query);
+    });
+
+    //Consumiendo servicios HEAD
+    it('Consume HEAD Service', async () => {
+        const response = await axios.head('https://httpbin.org/headers');
+
+        expect(response.status).to.equal(StatusCodes.OK)
+    });
+
+    //Consumiendo servicios DELETE
+    it('Consume DELETE Service', async () => {
+        const response = await axios.delete('https://httpbin.org/delete');
+
+        expect(response.status).to.equal(StatusCodes.OK);
+    });
+
+    //Consumiendo servicios POST
+    it('Consume POST Service', async () => {
+        const body = {
+            name: 'Fred',
+            age: '56',
+            city: 'New York'
+        }
+        const response = await axios.post('https://httpbin.org/post', { body });
+        expect(response.status).to.equal(StatusCodes.OK);
+    });
+
+    //Consumiendo servicios PATCH
+    it('Consume PATCH Service', async () => {
+        const body = {
+            name: 'Fred',
+            age: '56',
+            city: 'New York'
+        }
+        const response = await axios.patch('https://httpbin.org/patch', { body });
```

Comment:
you don't need to pass the body with the brackets, as the body that you are passing is already defined in json format.